android/build.gradle

```diff
@@ -25,5 +24,6 @@ allprojects {
         flatDir { dirs 'libs' }
         maven { url "http://139.162.11.12:8081/artifactory/libs-release-local" }
         maven { url "https://jitpack.io" }
+        google()
```

Comment:
?

Reply:
![image](https://user-images.githubusercontent.com/13368542/40550888-3bc3ddca-5ff0-11e8-814b-f4f3b9bbebb4.png)

Reply:
It was already there. Any reason to change its place?

Reply:
![image](https://user-images.githubusercontent.com/13368542/40554271-0a81a9ea-5ffa-11e8-95ac-9cd8379142db.png)